javascript-forms/form-values.js

```diff
@@ -0,0 +1,18 @@
+var contactForm = document.querySelector('#contact-form');
+var inputs = contactForm.elements;
+var values = {};
+
+contactForm.addEventListener('submit', function () {
+  event.preventDefault();
+  for (var i = 0; i < inputs.length; i++) {
+    if (inputs[i].name === 'name') {
+      values.name = inputs[i].value;
+    } else if (inputs[i].name === 'email') {
+      values.email = inputs[i].value;
+    } else if (inputs[i].name === 'message') {
+      values.message = inputs[i].value;
+      console.log(values);
+      contactForm.reset();
+    }
+  }
```

Comment:
You can do this without a for loop, the `elements` property has named keys for each input too, not just numbered indexes. You should log it to the console and take a look at its contents 🤓